From d1429c2349b80274255a676404d15bb2e25fa4b9 Mon Sep 17 00:00:00 2001
From: Varadarajan Narayanan <quic_varada@quicinc.com>
Date: Tue, 14 Mar 2023 11:43:34 +0530
Subject: [PATCH 04/41] soc: qcom: socinfo: Add IDs for IPQ9574 and its
 variants

Add SOC ID for Qualcomm IPQ9574, IPQ9570, IPQ9554, IPQ9550,
IPQ9514 and IPQ9510

Signed-off-by: Varadarajan Narayanan <quic_varada@quicinc.com>
Reviewed-by: Krzysztof Kozlowski <krzysztof.kozlowski@linaro.org>
Reviewed-by: Kathiravan T <quic_kathirav@quicinc.com>
Signed-off-by: Bjorn Andersson <andersson@kernel.org>
Link: https://lore.kernel.org/r/1678774414-14414-3-git-send-email-quic_varada@quicinc.com
---
 drivers/soc/qcom/socinfo.c | 6 ++++++
 1 file changed, 6 insertions(+)

--- a/drivers/soc/qcom/socinfo.c
+++ b/drivers/soc/qcom/socinfo.c
@@ -280,6 +280,12 @@ static const struct soc_id soc_id[] = {
 	{ 487, "SC7280" },
 	{ 495, "SC7180P" },
 	{ 507, "SM6375" },
+	{ 510, "IPQ9514" },
+	{ 511, "IPQ9550" },
+	{ 512, "IPQ9554" },
+	{ 513, "IPQ9570" },
+	{ 514, "IPQ9574" },
+	{ 521, "IPQ9510" },
 };
 
 static const char *socinfo_machine(struct device *dev, unsigned int id)
